v22.2/node-shutdown.md

```diff
@@ -800,6 +796,62 @@ If the decommissioning node has already reached the [draining stage](#draining),
 On the **Cluster Overview** page of the DB Console, the [node status](ui-cluster-overview-page.html#node-status) of the node should be `LIVE`. After a few minutes, you should see replicas rebalanced to the nodes.
 </section>
 
+## Decommissioning and draining on Kubernetes
+
+Most of the guidance in this page is most relevant to manual deployments that don't use Kubernetes. If you use Kubernetes to deploy CockroachDB, draining and decommissioning work the same way for the `cockroach` process, but Kubernetes handles them on your behalf. In a Kubernetes deployment, you don't usually decommission or drain a node interactively. In fact, if you attempt to do so, Kubernetes will automatically undo your change so that the cluster remains in its desired state as defined by the Kubernetes deployment. Instead, each cluster node is managed within Kubernetes.
+
+- Whether you deployed a cluster using the CockroachDB Operator, Helm, or a manual StatefulSet, the resulting deployment is a StatefulSet. Due to the nature of StatefulSets, it's safe to decommission **only** the Cockroach node with the highest StatefulSet ordinal in preparation for scaling down the StatefulSet. If you think you need to decommission any other node, consider the following recommendations and [contact Support](https://support.cockroachlabs.com/hc/en-us) for assistance.
+
+    - If you deployed a cluster using the [CockroachDB Kubernetes Operator](deploy-cockroachdb-with-kubernetes.html), there is no advantage to manually decommissioning the node with the highest StatefulSet ordinal. Instead, update the specification for the Kubernetes deployment to reduce the value of `nodes:` and apply the change using a [rolling update](https://kubernetes.io/docs/tutorials/kubernetes-basics/update/update-intro/). Kubernetes will notice that there are now too many nodes and will reduce them and clean up their storage automatically.
```

Comment:
I'm still confused by the use of "manually" to describe decommissioning in K8s, since the intro paragraph here mentions that "you don't usually decommission or drain a node interactively ... Instead, each cluster node is managed within Kubernetes."

Can you clarify what "manually" means in this context?

Reply:
Yes, you would not SSH into the Kubernetes node and issue a SIGTERM to the `cockroach` process. Instead. Kubernetes notices that the cluster now has too many nodes and selects one of them to send SIGTERM to. The cockroach process receives the SIGTERM and does the same thing it would have done if a sysadmin issued it manually.